### ISSUE TITLE
No connection to remote server

### ISSUE THREAD

**Issue by sergshabal** [sfackler/rust-postgres#192]
Hi!
Cant connect to remote postgresql 9.0 with this error:
`thread '<main>' panicked at 'no connection
Db(DbError { severity: "ERROR", code: UndefinedColumn, message: "column \"enumsortorder\" does not exist", detail: None, hint: None, position: Some(Normal(72)), where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: ".\\src\\backend\\parser\\parse_expr.c", line: 760, routine: "transformColumnRef" })', src\main.rs:14`

rust code:

```
extern crate postgres;
use postgres::{Connection, SslMode};
fn main() {
    let c1 = "postgresql://user:0000@10.10.98.4:5432/monitoring";
    let conn = match Connection::connect(c1, SslMode::None) {
            Ok(c) => {println!("connection Ok");c},
            Err(e) => panic!("no connection\n{:?}", e),
        };
}
```

but psql and pgadmin connection ok.
Where my mistake?

**Comment by sfackler** [comment]
Oops, looks like I need to add some extra handling for 9.0 - I should have a fix done tonight.

**Comment by sfackler** [comment]
I pushed a commit that should fix this - can you try it out? I don't have easy access to Postgres 9.0. https://github.com/sfackler/rust-postgres/commit/8e22dfba57d5181eddffca83913c8f2832b18309

**Comment by sergshabal** [comment]
its working. thank you!

**Comment by sfackler** [comment]
Great! I released v0.11.10 with this fix.
